lib/generation/generators/value_objects/generation_configuration/pb_generation_configuration.dart

```diff
@@ -139,12 +140,13 @@ abstract class GenerationConfiguration with PBPlatformOrientationGeneration {
       // _commitImports(tree.rootNode, tree.name.snakeCase, fileName);
 
       if (poLinker.screenHasMultiplePlatforms(tree.rootNode.name)) {
+        getPlatformOrientationName(tree.rootNode);
         tree.rootNode.currentContext.project.genProjectData.commandQueue
             .add(ExportPlatformCommand(
           tree.rootNode.currentContext.tree.data.platform,
           '$fileName',
-          '$fileName${getPlatformOrientationName(tree.rootNode)}.dart',
-          generationManager.generate(tree.rootNode),
+          '${tree.rootNode.name.snakeCase}.dart',
+          await generationManager.generate(tree.rootNode),
```

Comment:
Right now I dont think is an async method, not sure if we should turn `generate()` into an async method? 🤔

Reply:
Sure, let me try it